tests/introspection/ApiCoreImageFiltersTest.cs

```diff
@@ -65,26 +65,6 @@ protected virtual bool Skip (string nativeName)
 			// Both reported in radar #21548819
 			//  NSUnknownKeyException [<CIDepthOfField 0x158586970> valueForUndefinedKey:]: this class is not key value coding-compliant for the key inputPoint2.
 			case "CIDepthOfField":
-			// NSUnknownKeyException [<CISunbeamsGenerator 0x1586d0810> valueForUndefinedKey:]: this class is not key value coding-compliant for the key inputCropAmount.
-			case "CISunbeamsGenerator":
```

Comment:
^ that one seems unrelated to xcode9 - please double check (git blame)

Reply:
@spouliot removed because we do have this filter: https://github.com/xamarin/xamarin-macios/blob/master/src/coreimage.cs#L4563-L4594 (agree unrelated but that was easy to clean since I removed a bunch of cases.